lib/cisco_node_utils/cmd_ref/span_session.yaml

```diff
@@ -0,0 +1,64 @@
+# SPAN session
+---
+_template:
+  nexus:
+    get_command: "show running monitor all"
+
+all_sessions:
+  multiple:
+  get_context: ~
+  get_value: '/^monitor session (\d+)$/'
+
+create:
+  set_value: 'monitor session %s'
```

Comment:
I would prefer that we use `Key-value style` wildcards instead of `Printf style` wildcards.  Also keeps it consistent with the other properties in this file.

`set_value: 'monitor session <id>`